### PR TITLE
JAMES-1683 getMailboxes should return roles in lower case

### PR DESCRIPTION
**Pull request by Laura-Royet**

### CODE REVIEW

**Comment by rouazana** [comment]
:+1:

**Comment by aduprat** [comment]
:+1:

File: server/protocols/jmap-integration-testing/src/test/java/org/apache/james/jmap/methods/GetMailboxesMethodTest.java [hunk 1]
```diff
@@ -342,4 +342,25 @@ public void getMailboxesShouldReturnMailboxesWithSortOrder() throws Exception {
...
+        String response = given()
+            .accept(ContentType.JSON)
+            .contentType(ContentType.JSON)
+            .header("Authorization", accessToken.serialize())
+            .body("[[\"getMailboxes\", {}, \"#0\"]]")
+        .when()
+            .post("/jmap")
+        .then()
+            .statusCode(200)
+            .content(startsWith("[[\"mailboxes\","))
```
Comment (fvn-linagora) [inline server/protocols/jmap-integration-testing/src/test/java/org/apache/james/jmap/methods/GetMailboxesMethodTest.java]: use of "inline" json assertions (using .body()) would be more readable and less fragile (startsWith, json formatters and white spaces, risky bet!).

**Comment by fvn-linagora** [comment]
:+1:

**Comment by aduprat** [comment]
merged

**Pull request closed without merging**